app/Helpers/Helper.php

```diff
@@ -55,3 +57,15 @@ function userTime(null|Carbon|\Carbon\Carbon|string $time=null, ?string $format=
     }
     return $time->tz($timezone)->format($format);
 }
+
+function stationBoardTimezoneOffset(Collection $departures): bool {
```

Comment:
Nitpick: Since the function returns a bool, I'd like to rename it to `hasStationBoardTimezoneOffsetToUser` and let it receive `auth()->user()` as a parameter. This also makes it easier to test, as you can just insert a minimal `App\Models\User` object instead of mocking stuff.